addons/mysql/templates/_helpers.tpl

```diff
@@ -123,4 +123,295 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 
 {{- define "mysql.imagePullPolicy" -}}
 {{ default "IfNotPresent" .Values.image.pullPolicy }}
+{{- end }}
+
+{{- define "mysql.spec.common" -}}
```

Comment:
seems this template is used in both cmpd 5.7 and cmpd 8.0, udpate service version accordingly.